spec/index.html

```diff
@@ -849,51 +874,28 @@ <h3>Blank Nodes</h3>
   </section>
 
   <section id="section-triple-terms">
-    <h3>Triple Terms</h3>
+    <h3>Triple terms</h3>
```

Comment:
```suggestion
    <h3>Triple Terms</h3>
```
To be consistent with the way the other section titles are capitalized.